byterun/pyobj.py

```diff
@@ -135,8 +140,10 @@ def set(self, value):
 
 
 class Frame(object):
-    def __init__(self, f_code, f_globals, f_locals, f_back):
+    def __init__(self, f_code, f_globals, f_locals, f_closure, f_back):
         self.f_code = f_code
+        self.py36_opcodes = list(dis.get_instructions(self.f_code)) \
```

Comment:
Is there a reason to prefer this over just reading the f_code? If anything, 3.6 makes this very simple with the fixed width instructions.

Reply:
Given that Python is now exposing the API to iterate, and given that the bytecode format is not considered by Python project to be part of their public API, I think we should use their public API rather than read and interpret f_code.